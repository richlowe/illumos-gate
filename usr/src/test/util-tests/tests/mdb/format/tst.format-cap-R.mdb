1=RRRR
1<<0t62=RRR
1<<0t63=RRR
-1=RRRR
