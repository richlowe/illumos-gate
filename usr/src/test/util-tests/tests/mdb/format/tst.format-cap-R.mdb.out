                1
                1
                1
                1
                100000000000000000000000000000000000000000000000000000000000000
                100000000000000000000000000000000000000000000000000000000000000
                100000000000000000000000000000000000000000000000000000000000000
                1000000000000000000000000000000000000000000000000000000000000000
                1000000000000000000000000000000000000000000000000000000000000000
                1000000000000000000000000000000000000000000000000000000000000000
                1111111111111111111111111111111111111111111111111111111111111111
                1111111111111111111111111111111111111111111111111111111111111111
                1111111111111111111111111111111111111111111111111111111111111111
                1111111111111111111111111111111111111111111111111111111111111111
